### ISSUE TITLE
Would it be possible to display quarter-hour increments on the schedule?

### ISSUE THREAD

**Issue by mstem** [exafox/paquete#95]
So we can accept broadcast times that are 0:15 and 0:45 on the hour?

**Comment by jjandoc** [comment]
Just so I'm clear, the desired effect here is that we will have column headers at 15 min intervals? Or is it that we want the same column headers, but events can bisect the half hour increments when the even happens on the 15 and 45 min makrs?

**Comment by mstem** [comment]
Sorry, I should have been more clear - events can bisect the half hour
increments when the event happens on the 15 and 45 min markers

**Comment by mclaros** [comment]
The use of CSS grids in the TimeTable is pretty slick. I gotta read more about named grid areas. 

One dumb way I thought about approaching this was to calculate the percentage offset of the event to its column start time. We could then use that % as `left: X%` property, since each event cell appears to already be `position: relative`. For example, if an event is in the `5:30` column, but actually starts at 5:45, it could be `left: 50%` since 15 minutes is half of the 30 minute column intervals.

But if there's a css grid-based solution, that would be better. If we can assume we will only support quarter-hour intervals and not arbitrary times (like a start time 0:12), then can we set column widths at 15-minute intervals instead of 30?

**Comment by jjandoc** [comment]
> If we can assume we will only support quarter-hour intervals and not arbitrary times (like a start time 0:12), then can we set column widths at 15-minute intervals instead of 30?

Yeah, this is what I had in mind. The timeslot column headers should still be at 30 min intervals on the hour and half hour, but then can have a colspan of 2.
